src/main/java/frc/robot/autons/ShootInSpeaker.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.autons;
+
+import java.nio.file.Path;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import com.pathplanner.lib.path.PathPlannerPath;
+
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import frc.robot.commands.AcquireCommand;
+import frc.robot.commands.DriveVisionCommand;
+import frc.robot.commands.ScoringPositionCommand;
+import frc.robot.commands.ShootNoteCommand;
+import frc.robot.commands.StorageRunCommand;
+import frc.robot.subsystems.Scoring.ElevatorSetpoints;
+import frc.robot.subsystems.Vision.VisionTarget;
+
+public class ShootInSpeaker extends Auton {
+
+    public ShootInSpeaker(Alliance alliance) {
+        super(alliance);
+
+        PathPlannerPath startToShooting = PathPlannerPath.fromPathFile("From position 2 to speaker");
```

Comment:
these would all be trajectories and they would have their own separate file

https://github.com/LakotaRobotics1038/2023RobotProject/blob/main/src/main/java/frc/robot/autons/Trajectories.java